### ISSUE TITLE
frontend-build.zip is missing the .vite folder

### ISSUE THREAD

**Issue by Gigahawk** [inventree/InvenTree#7476]
### Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find a similar issue

### Describe the bug*

The precompiled frontend-build.zip file doesn't contain the vite manifest, causing the new react UI to not work properly

### Steps to Reproduce

Note I'm running inventree on NixOS via https://github.com/Gigahawk/nixos-inventree, but I don't think that should be related

1. Download https://github.com/inventree/InvenTree/releases/download/0.15.4/frontend-build.zip
2. It's missing `.vite/manifest.json`
3. Trying to navigate to `/platform` after extracting this zip results in `None` displayed in the browser. A `Manifest file not found` error is emitted to the logs
4. I've also tried manually generating the manifest file on a checkout of 0.15.4 but this results in a different set of asset files and therefore an incompatible manifest file.

### Expected behaviour

The frontend-build.zip file should contain the manifest file

### Deployment Method

- [ ] Docker
- [ ] Package
- [X] Bare metal
- [X] Other - added info in Steps to Reproduce

### Version Information

0.15.4

### Please verify if you can reproduce this bug on the demo site.

- [ ] I can reproduce this bug on the demo site.

### Relevant log output

```shell
Jun 19 03:02:52 virtualbox inventree-server[162382]: Manifest file not found
```

**Comment by Gigahawk** [comment]
Looks like this was partially fixed in #6973 but the actual release action wasn't included.

**Comment by Mixaill** [comment]
@SchrodingersGat 

Still not fixed, which causes `invoke update && invoke frontend-download --file` to show `None` on the `/platform` endpoint when using the master branch

![Image](https://github.com/user-attachments/assets/6be6145c-c095-46bb-93eb-239bd7c59000)
